scripts/TextExtractor.py

```diff
@@ -7,6 +7,7 @@
 from bs4 import BeautifulSoup
 import certifi
 from langchain_community.document_loaders import PyPDFLoader
+import logging
```

Comment:
here you are not using pep-8 for ordering the imports. logging as a build-in library should be before numpy, bs4 and others.